observability/logging/logging-6.2/log6x-configuring-lokistack-otlp-6.2.adoc

```diff
@@ -0,0 +1,145 @@
+= OTLP data ingestion in Loki
+include::_attributes/common-attributes.adoc[]
+:context: log6x-configuring-lokistack-otlp-6-2
+
+toc::[]
+
+Logging 6.2 enables an API endpoint using the OpenTelemetry Protocol (OTLP). As OTLP is a standardized format not specifically designed for Loki, it requires additional configuration on Loki's side to map OpenTelemetry's data format to Loki's data model. OTLP lacks concepts such as _stream labels_ or _structured metadata_. Instead, OTLP provides metadata about log entries as *attributes*, grouped into three categories:
+
+* Resource
+* Scope
+* Log
+
+This allows metadata to be set for multiple entries simultaneously or individually as needed.
+
+include::modules/log6x-configuring-lokistack-otlp-data-ingestion.adoc[leveloffset=+1]
+
+[id="attribute-mapping_{context}"]
+== Attribute mapping
+
+When the {loki-op} is set to `openshift-logging` mode, it automatically applies a default set of attribute mappings. These mappings align specific OTLP attributes with Loki's stream labels and structured metadata.
+
+For typical setups, these default mappings should be sufficient. However, you might need to customize attribute mapping in the following cases:
+
+* Using a custom Collector: If your setup includes a custom collector that generates additional attributes that you do not want to store, consider customizing the mapping to ensure these attributes are dropped by Loki.
+* Adjusting attribute detail levels: If the default attribute set is more detailed than necessary, you can reduce it to essential attributes only. This can avoid excessive data storage and streamline the {logging} process.
+
+[id="custom-attribute-mapping-for-openshift_{context}"]
+=== Custom attribute mapping for OpenShift
+
+When using the {loki-op} in `openshift-logging` mode, attribute mapping follow OpenShift defaults, but custom mappings can be configured to adjust these. Custom mappings allow further configurations to meet specific needs.
+
+In `openshift-logging` mode, custom attribute mappings can be configured globally for all tenants or for individual tenants as needed. When custom mappings are defined, they are appended to the OpenShift defaults. If default recommended labels are not required, they can be disabled in the tenant configuration.
+
+[NOTE]
+====
+A major difference between the {loki-op} and Loki itself lies in inheritance handling. Loki only copies `default_resource_attributes_as_index_labels` to tenants by default, while the {loki-op} applies the entire global configuration to each tenant in `openshift-logging` mode.
+====
+
+Within `LokiStack`, attribute mapping configuration is managed through the `limits` setting:
+
+[source,yaml]
+----
+# ...
+spec:
+  limits:
+    global:
+      otlp: {} # <1>
+    tenants:
+      application:
+        otlp: {} # <2>
+----
+<1> Global OTLP attribute configuration.
+<2> OTLP attribute configuration for the `application` tenant within `openshift-logging` mode.
```

Comment:
Not sure if it's relevant to say this
```suggestion
<2> OTLP attribute configuration for the `application` tenant within `openshift-logging` mode. `infrastructure` and `audit` are also supported
```

Reply:
@JoaoBraveCoding, how about:

2> OTLP attribute configuration for the `application`, `infrastructure`, or `audit` tenant within `openshift-logging` mode.